pkg/scheduler/framework/plugins/nodeaffinity/node_affinity_test.go

```diff
@@ -1006,6 +1006,10 @@ func TestNodeAffinityPriority(t *testing.T) {
 			if err != nil {
 				t.Fatalf("Creating plugin: %v", err)
 			}
+			status := p.(framework.PreScorePlugin).PreScore(context.Background(), state, test.pod, test.nodes)
```

Comment:
Add a test were PreScore is not called. No need to have a test table for it.

Reply:
Thanks @alculquicondor !
The test case has been added.